src/components/SchedulesPage/AllDay.tsx

```diff
@@ -36,11 +38,24 @@ interface Props {
 
 function AllDay({ day, nowDate, setNowDate, clickedDate, setClickedDate }: Props) {
   const Container = clsx('w-full flex justify-center items-center ');
-  const DateDay = clsx('text-black text-[1.2rem] font-bold w-[22rem] h-[15rem] text-center', {
-    'bg-[#EDEDED]':
-      day.getDay() === 0 || day.getDay() === 2 || day.getDay() === 4 || day.getDay() === 6, // 월, 수, 금
-    'bg-[#FFF]': day.getDay() === 1 || day.getDay() === 3 || day.getDay() === 5, // 화, 목, 토
-  });
+  const hover = clsx('hover:bg-[#D1D5DB]');
+  const DateDay = clsx(
+    'text-black text-[1.2rem] font-bold w-[22rem] h-[15rem] text-center',
+    {
+      'bg-[#EDEDED]':
+        day.getDay() === 0 || day.getDay() === 2 || day.getDay() === 4 || day.getDay() === 6, // 월, 수, 금
+      'bg-[#FFF]': day.getDay() === 1 || day.getDay() === 3 || day.getDay() === 5, // 화, 목, 토
+    },
+    hover,
+  );
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const openModal = () => {
+    setIsModalOpen(true);
+  };
+  const closeModal = () => {
+    setIsModalOpen(false);
+  };
+
```

Comment:
```suggestion
  const toggleModal = () => {
    setIsModalOpen(!isModalOpen);
  };
```
이거 저번 미팅 때 필겸님이 제 코드 리뷰해주신 것처럼 이렇게 함수 하나로 합칠 수 있을 것 같은데요?!

Reply:
넵! 감사합니다 ! 합쳐보겠습니다!